nservicebus/testing/index.md

```diff
@@ -18,12 +18,12 @@ To install this package:
 ```
 Install-Package NServiceBus.Testing
 ```
-Once the package is installed you need ensure that you call `Test.Initialize()` (or any of its overloads) before executing any test method.
+Once the package is installed you need ensure that you call `Test.Initialize()` (or any of its overloads) before executing any test method. This is **not needed **for Version 6.
```

Comment:
calling out that something is not needed in future versions at the end is kind of annoying. call it out before do people who keep up to date can skip the following text